samples/package-lock.json

```diff
@@ -1722,6 +2292,8 @@
     },
     "dialogflow": {
       "version": "0.2.1",
+      "resolved": "https://registry.npmjs.org/dialogflow/-/dialogflow-0.2.1.tgz",
```

Comment:
I guess this line is wrong as `dialogflow` must be taken from the parent directory (`npm link dialogflow` does it)